ddl/util/util.go

```diff
@@ -45,6 +46,9 @@ func (t DelRangeTask) Range() ([]byte, []byte) {
 func LoadDeleteRanges(ctx context.Context, safePoint uint64) (ranges []DelRangeTask, _ error) {
 	sql := fmt.Sprintf(loadDeleteRangeSQL, safePoint)
 	rss, err := ctx.(sqlexec.SQLExecutor).Execute(goctx.TODO(), sql)
+	if len(rss) > 0 {
+		defer terror.Call(rss[0].Close)
```

Comment:
maybe this is more safe:
```go
defer func() {
    for _, rs := range rss {
        terror.Call(rs.Close)
    }
}()
```

Reply:
We can make sure we only execute one statement.

Reply:
rss can contain at most 1 recordSet, the code block below also depends on this constraint.

Reply:
OK